src/Lecture3.hs

```diff
@@ -82,7 +83,8 @@ Tuesday
   would work for **any** enumeration type in Haskell (e.g. 'Bool',
   'Ordering') and not just 'Weekday'?
 -}
-next = error "TODO"
+next :: (Enum a) => a -> a
+next = succ
```

Comment:
This is almost correct 👌🏻 
Unfortunately, `succ` throws an error on the last constructor of the data type